crates/meilisearch/tests/search/multi/mod.rs

```diff
@@ -3604,22 +3604,28 @@ async fn federation_non_faceted_for_an_index() {
 
     let index = server.index("fruits");
 
+    let documents = FRUITS_DOCUMENTS.clone();
+    let (value, _) = index.add_documents(documents, None).await;
+    index.wait_task(value.uid()).await.succeeded();
```

Comment:
this change shouldn't be necessary anymore, now that we display the filterable *patterns* in error messages and that we compute the max weight regardless of which fields actually exist in the dataset.

Should we consider undoing the changes where this PR is adding documents to indexes?